src/scenes/home/Home.js

```diff
@@ -80,7 +98,7 @@ export class Home extends Component {
           tasks={tasks}
           toggleDone={this.toggleDone}
           onPressTask={id => {
-            const t = data.find(x => x.id === id);
+            const t = tasks.find(x => x.id === id);
```

Comment:
Same here always avoid put arrow functions inside jsx, its for performance issues